.github/workflows/push-to-dokku.yml

```
@@ -0,0 +1,25 @@
+name: Push to Dokku
+
+on:
+  workflow_call:
+    inputs:
+      remote_url:
+        required: true
+        type: string
+        description: "The remote URL to push to"
+      image_name:
+        required: true
+        type: string
+        description: "The name of the image to push"
+
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Push
+        uses: dokku/github-action@v1.4.0
+        with:
+          git_remote_url: ${{ inputs.remote_url }}
+          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
+          deploy_docker_image: ${{ inputs.image_name }}
```

```suggestion
          git_remote_url: ${{ inputs.git_remote_url }}
          ssh_private_key: ${{ secrets.SSH_PRIVATE_KEY }}
          deploy_docker_image: ${{ inputs. deploy_docker_image }}
```

Matching input names to actual args will make the workflow easier to understand.